examples/examples.js

```diff
@@ -24,6 +24,7 @@ var categories = [
             "layers",
             "lights",
             "lights-baked",
+            "lights-baked-render",
```

Comment:
I would just make this `lights-baked` and put the previous one in a `deprecated` folder.